build-image-src/build_all_images.sh

```diff
@@ -39,8 +39,7 @@ docker build -f Dockerfile-python39 -t amazon/aws-sam-cli-build-image-python3.9:
 docker build -f Dockerfile-ruby27 -t amazon/aws-sam-cli-build-image-ruby2.7:x86_64 --platform linux/amd64 --build-arg SAM_CLI_VERSION=$SAM_CLI_VERSION --build-arg AWS_CLI_ARCH=x86_64 --build-arg IMAGE_ARCH=x86_64 . &
 wait
 
-# Delete multi arch al2 image and start building arm64 images
-docker rmi public.ecr.aws/amazonlinux/amazonlinux:2
```

Comment:
Same comment as previous, did we confirm that it works without this `docker rmi`?

Reply:
Answered in the other comment